features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -136,6 +136,7 @@
     </capability>
 
     <bundle>mvn:org.openhab.core.bundles/org.openhab.core.voice/${project.version}</bundle>
+    <bundle>mvn:org.openhab.core.bundles/org.openhab.core.io.bin2json/${project.version}</bundle>
```

Comment:
It should not be added to base (openhab-core-base) feature.

If you would like to ensure that it is present in every openHAB distribution, then add it to `openhab-runtime-base`.

I would prefer a separate feature for bin2json at all that references the specific TP feature (see below).